.travis.yml

```diff
@@ -56,7 +61,9 @@ script:
   # Move to the path of the installed version
   - cd $(python -c 'import theano; import os; print(os.path.split(theano.__file__)[0])')
   - echo "$PART"
-  - theano-nose -v $PART
+  - cd -; cd Theano
+  - if [[ $DOC == "1" ]]; then python doc/scripts/docgen.py --nopdf; else theano-nose -v $PART; fi
+  - if [[ $DOC == "1" ]]; then python doc/scripts/docgen.py --test; else theano-nose -v $PART; fi
```

Comment:
this trigger 2 time Theano tests when not in DOC mode. Just remove the else part on this line.